src/sqlfluff/testing/rules.py

```diff
@@ -87,11 +87,14 @@ def assert_rule_pass_in_sql(code, sql, configs=None):
     # Configs allows overrides if we want to use them.
     cfg = FluffConfig(configs=configs)
     r = get_rule_from_set(code, config=cfg)
-    parsed = Linter(config=cfg).parse_string(sql)
+    templated_file = []
+    parsed = Linter(config=cfg).parse_string(sql, templated_file=templated_file)
```

Comment:
Previously, rule tests using `pass_str` didn't receive a `templated_file` parameter, which is now needed by L001.